build.sbt

```diff
@@ -1,30 +1,25 @@
-import java.io.File
-import java.net.{URL, URLEncoder}
+import java.io.{File, PrintWriter}
+import java.net.URL
 
-import org.apache.commons.io.{FileUtils, IOUtils}
+import org.apache.commons.io.FileUtils
 import sbt.internal.util.ManagedLogger
 
 import scala.sys.process.Process
-import sys.process._
-import java.net.URL
-import java.io.File
 
 val condaEnvName = "mmlspark"
 name := "mmlspark"
 organization := "com.microsoft.ml.spark"
 scalaVersion := "2.11.12"
 
 val sparkVersion = "2.4.3"
-val sprayVersion = "1.3.4"
-val cntkVersion = "2.4"
 
 libraryDependencies ++= Seq(
   "org.apache.spark" %% "spark-core" % sparkVersion % "compile",
   "org.apache.spark" %% "spark-mllib" % sparkVersion % "compile",
   "org.scalactic" %% "scalactic" % "3.0.5",
   "org.scalatest" %% "scalatest" % "3.0.5",
   "io.spray" %% "spray-json" % "1.3.2",
-  "com.microsoft.cntk" % "cntk" % cntkVersion,
+  "com.microsoft.cntk" % "cntk" % "2.4",
```

Comment:
Why is this change needed? It seems useful to have all constants declared at the top?